pkg/tests/tasks/traffic/egress/access_external_services_test.go

```diff
@@ -15,102 +15,150 @@
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/env"
+	"github.com/maistra/maistra-test-tool/pkg/util/hack"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
 	"github.com/maistra/maistra-test-tool/pkg/util/test"
 )
 
-func cleanupAccessExternalServices() {
-	log.Log.Info("Cleanup")
-	sleep := examples.Sleep{Namespace: "bookinfo"}
-	util.KubeDeleteContents("bookinfo", httpbinextTimeout)
-	util.KubeDeleteContents("bookinfo", redhatextServiceEntry)
-	util.KubeDeleteContents("bookinfo", httpbinextServiceEntry)
-	sleep.Uninstall()
-	time.Sleep(time.Duration(20) * time.Second)
-}
-
 func TestAccessExternalServices(t *testing.T) {
-	test.NewTest(t).Id("T11").Groups(test.Full, test.InterOp).NotRefactoredYet()
-
-	defer cleanupAccessExternalServices()
-	defer util.RecoverPanic(t)
-
-	log.Log.Info("TestAccessExternalServices")
-	sleep := examples.Sleep{Namespace: "bookinfo"}
-	sleep.Install()
-	sleepPod, err := util.GetPodName("bookinfo", "app=sleep")
-	util.Inspect(err, "Failed to get sleep pod name", "", t)
-
-	t.Run("TrafficManagement_egress_envoy_passthrough_to_external_services", func(t *testing.T) {
-		defer util.RecoverPanic(t)
-
-		log.Log.Info("Skip checking the meshConfig outboundTrafficPolicy mode")
-		log.Log.Info("make requests to external https services")
-		command := `curl -sSI https://www.redhat.com/en | grep  "HTTP/"`
-		msg, err := util.PodExec("bookinfo", sleepPod, "sleep", command, false)
-		util.Inspect(err, "Failed to get response", "", t)
-		if strings.Contains(msg, "200") {
-			log.Log.Infof("Success. Get https://www.redhat.com/en response: %s", msg)
-		} else {
-			log.Log.Infof("Error response: %s", msg)
-			t.Errorf("Error response: %s", msg)
-		}
-	})
+	test.NewTest(t).Id("T11").Groups(test.Full, test.InterOp).Run(func(t test.TestHelper) {
+		hack.DisableLogrusForThisTest(t)
 
-	t.Run("TrafficManagement_egress_controlled_access_to_external_httpbin_services", func(t *testing.T) {
-		defer util.RecoverPanic(t)
-
-		log.Log.Info("Skip update global.outboundTrafficPolicy.mode")
-		log.Log.Info("Create a ServiceEntry to external httpbin")
-		util.KubeApplyContents("bookinfo", httpbinextServiceEntry)
-		time.Sleep(time.Duration(10) * time.Second)
-		command := `curl -sS http://httpbin.org/headers`
-		msg, err := util.PodExec("bookinfo", sleepPod, "sleep", command, false)
-		if err != nil {
-			log.Log.Infof("Error response: %s", msg)
-			t.Errorf("Error response: %s", msg)
-		} else {
-			log.Log.Infof("Success. Get http://httpbin.org/headers response:\n%s", msg)
-		}
-	})
+		ns := "bookinfo"
+		meshNamespace := env.GetDefaultMeshNamespace()
+		smcpName := env.GetDefaultSMCPName()
+		outboundDefaultPatch := `[{"op": "remove", "path": "/spec/proxy/networking/trafficControl/outbound/policy"}]`
+		t.Cleanup(func() {
+			app.Uninstall(t, app.Sleep(ns))
+			oc.Patch(
+				t,
+				meshNamespace,
+				"smcp",
+				smcpName,
+				"json",
+				outboundDefaultPatch,
+			)
```

Comment:
nit:I'd format this a bit differently:
```suggestion
			oc.Patch(t,
				meshNamespace,
				"smcp", smcpName,
				"json", outboundDefaultPatch)
```